src/python/pants/build_graph/address_test.py

```diff
@@ -383,34 +383,6 @@ def assert_noops(addr: Address) -> None:
     assert_noops(Address("a/b"))
 
 
-def test_address_maybe_convert_to_generated_target() -> None:
```

Comment:
WhY?

Reply:
The method was unused outside of its test: the method was also removed.